app/assets/javascripts/discourse/app/services/composer.js

```diff
@@ -1041,13 +1042,21 @@ export default class ComposerService extends Service {
     }
 
     const composer = this.model;
+    const cantSubmitPost = applyValueTransformer(
+      "composer-service-cant-submit-post",
```

Comment:
```suggestion
      "composer-service-cannot-submit-post",
```

Purely a suggestion, only because I can't process "cant" without an apostrophe!